test/test_node.py

```diff
@@ -17,7 +17,7 @@ def test_inputs_are_cut_to_match_arity():
     node = cgp.ConstantFloat(idx, inputs)
     assert node.inputs == []
 
-    node = cgp.node.OutputNode(idx, inputs)
+    node = cgp.node_input_output.OutputNode(idx, inputs)
```

Comment:
I would either directly import `OutputNode` and `InputNode` in `cgp.__init__.py` or rename the the file to `node_io`.

Reply:
as this is no user-facing code (users never create instances of `Input/OutputNode`s) i would actually leave this ugly, but clear module name and avoid the unnecessary import in `__init__`. are there other good reasons that i am missing?

Reply:
Okay, that's okay. I would still prefer to rename the file to `node_io.py`, since `io` is a common abbreviation for `input_output`. But it's a matter of taste, I guess.